cmd/juju/controller/showcontroller.go

```diff
@@ -273,7 +274,7 @@ type MachineDetails struct {
 	ID string `yaml:"id,omitempty" json:"id,omitempty"`
 
 	// InstanceID holds the cloud instance id of the machine.
-	InstanceID string `yaml:"instance-id,omitempty" json:"instance-id,omitempty"`
+	InstanceID instance.Id `yaml:"instance-id,omitempty" json:"instance-id,omitempty"`
```

Comment:
This is purely for output purposes so no need for anything other than a string

Reply:
Sorted.